keras_hub/src/utils/keras_utils.py

```diff
@@ -72,3 +72,9 @@ def has_flash_attention_support():
         return True
     else:
         return False
+
+
+def running_on_tpu():
```

Comment:
If we are going to name this as is, let's make this work on any backend, otherwise this is a silly gotcha.

```python
if backend == "jax":
    import jax

    return any(d.platform == "tpu" for d in devices)
elif backend == "tensorflow":
    import tensorflow as tf

    return bool(tf.config.list_logical_devices("TPU"))
elif backend == "torch":
    return False
```

Reply:
So if we add a `running_on_gpu` that can basically look the same. `torch.cuda.is_available()` for torch for that one.